src/tests/TestACLUserRegistration/TestACLUserRegistration.cpp

```
@@ -0,0 +1,71 @@
+// The ACL editor determines whether a user is registered by checking
+// whether their user ID is non-negative. A user ID of -1 indicates an
+// unregistered (guest) user, while a user ID >= 0 indicates a registered
+// user. This is the same logic used in ACLEditor.cpp to decide whether
+// to show a warning when an administrator adds a user to an ACL.
+
+// See: src/mumble/ACLEditor.cpp (on_qcbACLUser_textActivated)
+// See: src/User.h (iId field)
+
+class TestACLUserRegistration : public QObject {
+	Q_OBJECT
+private slots:
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)